Cubical/Algebra/Monoid/BigOp.agda

```diff
@@ -57,3 +57,17 @@ module MonoidBigOp (M' : Monoid ℓ) where
 
  bigOpSplit++ {n = zero} V W = sym (·IdL _)
  bigOpSplit++ {n = suc n} V W = cong (V zero ·_) (bigOpSplit++ (V ∘ suc) W) ∙ ·Assoc _ _ _
+
+
+module _ {M : Monoid ℓ} {M' : Monoid ℓ'} (φ : MonoidHom M M') where
+  module M = MonoidBigOp M
+  module M' = MonoidBigOp M'
```

Comment:
These two should probably be private. (`module ... = ...` statements are public by default, as opposed to `open ...` statements.)